examples/README.md

```diff
@@ -3,6 +3,12 @@ Optuna Examples
 
 This page contains a list of example codes written with Optuna.
 
+You can use our docker images with the tag ending with `-dev` to run most of the examples.
+For example, you can run [PyTorch Lightning Simple](./pytorch_ignite_simple.py) via `docker run --rm optuna/optuna:py3.7-dev python pytorch_lightning_simple.py`.  
+Also, you can try our visualization example in Jupyter Notebook by opening `localhost:8888` in your browser after executing this:
+```bash
+docker run -p 8888:8888 --rm optuna/optuna:py3.7-dev jupyter notebook --allow-root --no-browser --port 8888 --ip 0.0.0.0 --NotebookApp.token='' --NotebookApp.password=''
+
```

Comment:
Great initiative in helping out a broader range of users. I'm just not sure whether we should put this info near the top. I might be overthinking, but it could be overwhelming for some, and might give an impression that Optuna is rather complex to run. How about leaving a separate section after the list of examples with e.g. "Struggling with setting up?" or "Running with Optuna's Docker Images"?

Reply:
So, where do you want that section to be placed?

Reply:
How about at the bottom, with h2?